### ISSUE TITLE
Interesting!

### ISSUE THREAD

**Issue by richard-uk1** [willcrichton/so-lang-diversity-analysis#1]
Do you have any hypotheses for why you see the variation?

**Comment by willcrichton** [comment]
Not really. I don't know many of the language communities (e.g. APL) well enough to make a confident claim about how they attract members. But it's a good question for someone to investigate.
